import-data.sh

```diff
@@ -44,6 +44,12 @@ ls -al $WORKING_DIR
 ls -al $DATA_DIR
 ls -al $TEAM_IMAGES_DIR
 
+# Check if dataversion exists
+DATAVERSION_PATH="${DATA_DIR}/dataversion.json"
+if [ ! -f "${DATAVERSION_PATH}" ]; then
+  DATAVERSION_PATH="${WORKING_DIR}/data-manifest.json"
+fi
+
```

Comment:
To help debugging I would print a message with the `DATAVERSION_PATH` to let you know which one is being used before import happens.